pkg/lib/hook/deno_client.go

```diff
@@ -93,16 +95,27 @@ func (c *DenoClientImpl) Run(ctx context.Context, snippet string, input interfac
 
 	logger.With(
 		slog.String("response_error", runResponse.Error),
+		slog.String("error_code", string(runResponse.ErrorCode)),
 		slog.Any("output", runResponse.Output),
 		slog.Any("stdout", runResponse.Stdout),
 		slog.Any("stderr", runResponse.Stderr),
 	).Info(ctx, "run deno script")
 
 	if runResponse.Error != "" {
+		otelutil.IntCounterAddOne(ctx,
+			otelauthgear.CounterDenoRunCount,
+			otelauthgear.WithStatusError(),
+			otelauthgear.WithDenoErrorCode(string(runResponse.ErrorCode)),
+		)
+
```

Comment:
As mentioned, the core limits requests to 5s and 60s for blocking and non-blocking hooks. I think we should also track these as timeout cases. And this is even more useful because these timeouts are shorter or equal to Deno’s timeout, which means we may not have many opportunities to trigger Deno’s own timeout in practice.